lib/active_model/uninitialized_sobject.rb

```diff
@@ -0,0 +1,12 @@
+require 'active_model'
+
+module ActiveModel
+  class Attribute
+    class UninitializedSobject < Uninitialized # :nodoc:
```

Comment:
Not loving the name of this class. Maybe `UninitializedValue` makes more sense? I dunno.

Reply:
Good call and I agree. It's more like `UninitializedSobjectAttribute`; but that is so lengthy. I'll change it to `UninitializedValue`